src/tiledb/cloud/bioimg/exportation.py

```diff
@@ -51,49 +100,69 @@ def export_tiff_udf(
         :param io_uris: Pairs of tiff input - output tdb uris
         :param config: dict configuration to pass on tiledb.VFS
         """
+        from tiledb.bioimg import Converters
+        from tiledb.bioimg import to_bioimg
+        from tiledb.ctx import default_ctx
 
-        from tiledb.bioimg.converters.ome_tiff import OMETiffConverter
+        # if writer config not given assume same as source
+        src_cfg = default_ctx().config()
+        dest_cfg = src_cfg if not config else config
 
-        conf = tiledb.Config(params=config)
         for input, output in io_uris:
-            OMETiffConverter.from_tiledb(input, output, config=conf, **kwargs)
+            to_bioimg(
+                input,
+                output,
+                converter=Converters.OMETIFF,
+                config=src_cfg,
+                output_config=dest_cfg,
+                verbose=verbose,
+                **kwargs,
+            )
 
     if isinstance(source, str):
         # Handle only lists
         source = [source]
 
-    # Get the list of all BioImg samples input/out
-    samples = get_uris(source, output, config, "tiff")
-    batch_size, max_workers = scale_calc(samples, num_batches)
-
     # Build the task graph
     dag_name = taskgraph_name or DEFAULT_DAG_NAME
-    task_prefix = f"{dag_name} - Task"
 
-    logger.info("Building graph")
-    graph = tiledb.cloud.dag.DAG(
+    logger.debug("Building graph")
+    graph = dag.DAG(
         name=dag_name,
-        mode=tiledb.cloud.dag.Mode.BATCH,
+        mode=dag.Mode.REALTIME if local else dag.Mode.BATCH,
         max_workers=max_workers,
         namespace=namespace,
     )
 
-    for i, work in enumerate(batch(samples, batch_size)):
-        logger.info(f"Adding batch {i}")
-        graph.submit(
-            export_tiff_udf,
-            work,
-            config,
-            *args,
-            name=f"{task_prefix} - {i}",
-            mode=tiledb.cloud.dag.Mode.BATCH,
-            resources=DEFAULT_RESOURCES if resources is None else resources,
-            image_name=DEFAULT_IMG_NAME,
-            **kwargs,
-        )
+    # The lister doesn't need many resources.
+    input_list_node = graph.submit(
+        build_input_batches,
+        source,
+        output,
+        num_batches,
+        output_ext,
+        access_credentials_name=kwargs.get("access_credentials_name"),
+        name=f"{dag_name} input collector",
+        result_format="json",
+    )
+
+    logger.debug("Batched Input-Output pairs: %s", input_list_node)
+
+    graph.submit(
+        export_tiff_udf,
+        input_list_node,
+        config,
+        verbose,
+        *args,
+        name=f"{dag_name} exporter",
+        expand_node_output=input_list_node,
+        resources=DEFAULT_RESOURCES if resources is None else resources,
```

Comment:
likewise, `resources or DEFAULT_RESOURCES` (although this would also use `DEFAULT_RESOURCES` if `resources` were `{}`, so `is None` may be the appropriate check here)